kolibri/core/device/utils.py

```diff
@@ -90,13 +90,13 @@ def set_device_settings(**kwargs):
         raise DeviceNotProvisioned
 
 
-def provision_device(device_name=None, **kwargs):
+def provision_device(device_name=None, is_provisioned=True, **kwargs):
```

Comment:
If `is_provisioned` is not set on the request to the serializer, then `is_provisioned` here will be `None`, not `True` - the default value is only used in the case that the value is not passed to the function at all.